07_errors/patrickmarabeas/types.go

```diff
@@ -1,21 +1,19 @@
 package main
 
-import (
-	"sync"
-)
+import "sync"
 
 type Puppy struct {
 	ID    int
 	Breed string
 	Color string
-	Value string
+	Value int // cents
 }
 
 type Storer interface {
-	Create(puppy Puppy) int
-	Read(ID int) Puppy
-	Update(ID int, puppy Puppy) bool
-	Destroy(ID int) bool
+	Create(puppy Puppy) (int, error)
+	Read(ID int) (Puppy, error)
+	Update(ID int, puppy Puppy) (bool, error)
```

Comment:
Since the return value of `Update()` was an error of sorts before, does it makes sense to keep the bool here? Just return `nil` or an error perhaps.